generators/client/templates/angular/src/main/webapp/app/admin/metrics/metrics-modal-threads/metrics-modal-threads.component.html.ejs

```diff
@@ -50,8 +50,8 @@
         </span>
     </div>
 
-    <div class="pad" *ngFor="let thread of getThreads()">
-        <h6>
+    <div id="modalThreadsMetrics" class="pad" *ngFor="let thread of getThreads()">
+        <h6 id="modalThreadsMetrics{{ thread.threadId }}">
             <span class="badge" [ngClass]="getBadgeClass(thread.threadState)">{{ thread.threadState }}</span>
 
             &nbsp;{{ thread.threadName }} (ID {{ thread.threadId }})
```

Comment:
Here `id="modalThreadsMetrics"` is not best reference for `aria-describedby`.  
`id="modalThreadsMetrics{{ thread.threadId }}"` is unused.  
Maybe put `{{ thread.threadName }}` into `span` and give it id which includes `{{ thread.threadId }}` and in table use the same id in `aria-describedby` attribute?

Reply:
One possible solution is to follow Sonar instructions:

```
In order to be accessible to visually impaired users, it is important that tables provides a description of its content before the data is accessed.

The simplest way to do it, and also the one recommended by WCAG2 is to add a <caption> element inside the <table>.
```

Reply:
done